atmdemo/index.php

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>ATM demo</title>
+</head>
+<body>
+    
+    <div class="container">
+      
+      <div class="row align-items-start">
+          
+          <div class="col-xs-4 col-sm-4 col-md-4 col-lg-4">
+              <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQAm3nofxSF-qp_-ubpgT_UWZPsAmjg1xsiz5p_dSUHMokUGZ-r042QBMl9ERLM_bHcz-k&usqp=CAU">
+          </div>
+          
+          <div class="col-xs-8 col-sm-8 col-md-8 col-lg-8">
+              <p>ATM demo</p>
+              <p>nhập vào số tiền quý khách muốn rút </p>
+              <?php
+                  if(isset($_POST["moneyinput"])) {
+                      $money = $_POST["moneyinput"];
+                      $flag = $money;
```

Comment:
ko dùng thì xóa thôi e

Reply:
cái này e khai báo dùng để in tổng tiền mà quên mất dùng cái khác :v